src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -212,7 +212,9 @@ public GenerateScriptCommand(OctoLogger log, ISourceGithubApiFactory sourceGithu
 
         private async Task<string> InvokeGithub(string githubSourceOrg, string githubTargetOrg, string ghesApiUrl, string azureStorageConnectionString, bool noSslVerify, bool sequential, string githubSourcePat, bool skipReleases)
         {
-            var repos = await GetGithubRepos(_sourceGithubApiFactory.Create(sourcePersonalAccessToken: githubSourcePat), githubSourceOrg);
+            var repos = string.IsNullOrWhiteSpace(ghesApiUrl) ? 
+                await GetGithubRepos(_sourceGithubApiFactory.Create(sourcePersonalAccessToken: githubSourcePat), githubSourceOrg) :
+                await GetGithubRepos(_sourceGithubApiFactory.Create(apiUrl: ghesApiUrl, sourcePersonalAccessToken: githubSourcePat), githubSourceOrg);
```

Comment:
On line 125 in this file you'll need to set `string ghesApiUrl = ""` to be `string ghesApiUrl = null`. Otherwise the `??=` operand in the create function will pick the empty string. `??=` only checks if the left hand side is null.

Reply:
thought of this but wasn't sure if it would break something else lol.

Reply:
When I tried it locally it broke things :)